.prout.json

```diff
@@ -3,14 +3,6 @@
         "PROD": {
             "url": "https://membership.theguardian.com/",
             "overdue": "14M",
```

Comment:
You need to nuke the comma! https://app.getsentry.com/the-guardian/prout/issues/112278760/